CRM/Core/BAO/FinancialTrxn.php

```diff
@@ -15,17 +15,6 @@
  * @copyright CiviCRM LLC https://civicrm.org/licensing
  */
 class CRM_Core_BAO_FinancialTrxn extends CRM_Financial_DAO_FinancialTrxn {
-  /**
-   * Class constructor.
-   *
-   * @return \CRM_Financial_DAO_FinancialTrxn
-   */
-
-  /**
```

Comment:
this will make @pradpnayak's PR stale - should be an easy tweak to that PR but it might come back depending on how the merge conflict fixing goes...

Reply:
This PR is probably easier to rebase if you wanna merge that one first.

Reply:
@colemanw #22400 is currently failed tests - I pulled out #22546 as a partial that renames the class & adds in the temporary placeholder class that extends the new 'real' class and removed the constructor. Hopefully that passes & has the changes from both PRs that touch this file (without having an answer on the failing tests at this stage)

Reply:
@colemanw this got merged in the meantime - turns out the test error @pradpnayak was hitting is still present in the smallest version of the patch & I can't run the test myself to see why